the-collab-lab/tcl-23-smart-shopping-list
# Tailwind app architecture and include food hero pattern

## Description

1. In `App.js` create Tailwind utility classes on `<main>` element for inherited colors, fonts and breakpoints shared by the entire project.
2. Remove the vanilla CSS from `App.css` and replace with food hero pattern in the `body {}` class.

## Related Issue



Closes #44 
Closes #36 

## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    ✓| :sparkles: New feature     |
|    ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

```javascript
 <main className="App">
```
![Screenshot (153)](https://user-images.githubusercontent.com/47455758/119270897-986a0080-bbc4-11eb-958c-630e97c43986.png)

### After

```javascript
<main className="flex flex-col items-center bg-midnight-green min-h-screen text-gray-200 font-roboto mx-auto lg:w-2/3">
```

![Screenshot (154)](https://user-images.githubusercontent.com/47455758/119270910-b0da1b00-bbc4-11eb-8554-780e9336436f.png)

## Testing Steps / QA Criteria

1. On main run `git pull`
2. Run `git checkout tailwind-app-architecture`
3. Run `git install`
4. Run `npm start`
5. Open your favorite responsive browser tool and observe the behaviour of the hero pattern hiding/showing at Tailwind's `lg` breakpoint which is `1024px` https://tailwindcss.com/docs/breakpoints
